src/routes/github/create-branch/github-create-branch-get.ts

```diff
@@ -113,9 +113,8 @@ const getReposBySubscriptions = async (subscriptions: Subscription[], logger: Lo
 			const response = await gitHubInstallationClient.getRepositoriesPage(100, undefined, "UPDATED_AT");
 			// The app can be installed in a GitHub org but that org might not be connected to Jira, therefore we must filter them out, or
 			// the next steps (e.g. get repo branches to branch of) will fail
-			const subscriptionRepos = await RepoSyncState.findAllFromSubscription(subscription);
-			const subscriptionRepoIds = new Set(subscriptionRepos.map(repo => repo.repoId));
-			const filteredRepos =  response.viewer.repositories.edges.filter(edge => subscriptionRepoIds.has(edge.node.id));
+			const repoOwners = await RepoSyncState.findAllRepoOwners(subscription);
```

Comment:
We don't need this? As the `createInstallationClient` is already for a particular `org`, no need to further filter

Reply:
We need because when we fetch branches, we pass the org as a parameter:
- https://github.com/atlassian/github-for-jira/blob/792dab5e01f35ac64a56b73a4228d021f0f82f06/src/routes/github/create-branch/github-branches-get.ts#L19-L19

If the app doesn't know about that org, the call will fail